patches/chrome-browser-upgrade_detector-upgrade_detector_impl.cc.patch

```diff
@@ -10,12 +10,12 @@ index a12c9d9958376c92e8c5a78d8ab8182d5ecafd77..f29ec211b7ae2bd2bbd520cb69ad662d
      } else {
        switch_name = switches::kSimulateOutdated;
      }
-@@ -477,7 +476,7 @@ void UpgradeDetectorImpl::Init() {
+@@ -476,7 +475,7 @@ void UpgradeDetectorImpl::Init() {
+ 
    // On Windows, only enable upgrade notifications for Google Chrome builds.
    // Chromium does not use an auto-updater.
- #if !BUILDFLAG(IS_WIN) || BUILDFLAG(GOOGLE_CHROME_BRANDING) || \
--    BUILDFLAG(ENABLE_CHROMIUM_UPDATER)
-+    BUILDFLAG(ENABLE_CHROMIUM_UPDATER) || defined(OFFICIAL_BUILD)
+-#if !BUILDFLAG(IS_WIN) || BUILDFLAG(GOOGLE_CHROME_BRANDING)
```

Comment:
I think maybe we just want to set `GOOGLE_CHROME_BRANDING` to true for this file? Don't we also want `ShouldDetectOutdatedBuilds` to return true?